config/install/replication.replication_settings.entity_type.article.yml

```diff
@@ -0,0 +1,9 @@
+langcode: en
```

Comment:
I find the ID of this entity confusing. Why is it entity_type.article? Shouldn't it maybe be node.article...?